app/javascript/stylesheets/application.css

```diff
@@ -8,31 +8,24 @@
     @apply text-lg md:text-xl xl:text-2xl;
   }
 
-  h1 {
+  .header-1 {
     @apply text-6xl md:text-7xl 2xl:text-8xl font-bold !important;
   }
 
-  h2 {
+  .header-2 {
     @apply text-4xl md:text-5xl 2xl:text-6xl font-medium !important;
   }
 
-  h3,
-  h4 {
+  .header-3-4 {
     @apply text-3xl md:text-4xl 2xl:text-5xl font-bold mb-8 !important;
   }
```

Comment:
Just one observation on modals, styles were not modified, they are located in `config/locales/es.yml`:

![imagen](https://user-images.githubusercontent.com/12243331/108447472-88d5c700-7225-11eb-9d21-a20f52f9d314.png)

Reply:
Done 👍